### PR TITLE
add a direct install link for distros other than arch

### PR DESCRIPTION
**Pull request by lwertzui**
the link just installs without having to seperately having to install and run it

### CODE REVIEW

**Comment by lwertzui** [comment]
kinda embarassing i needed 3 commits to correct all spelling mistakes i made in the first one, but it is 3am atm i guess

**Comment by n3oney** [comment]
smart, but please link to my repo and not your fork :3

**Comment by BBaoVanC** [comment]
I would personally recommend a pipe (`curl | bash`) instead of using process substitution since it's simpler and even POSIX complaint

**Comment by lwertzui** [comment]
sorry bout that, fixed now merge plz?

**Comment by n3oney** [comment]
> I would personally recommend a pipe (`curl | bash`) instead of using process substitution since it's simpler and even POSIX complaint

++

**Comment by lwertzui** [comment]
added that while you were typing

**Comment by n3oney** [comment]
just one small thing, maybe mention that the script depends on JQ?

**Comment by lwertzui** [comment]
i think its enough to just say it in the dependecies right?

**Comment by lwertzui** [comment]
oh your script doesnt check dependecies

**Comment by lwertzui** [comment]
one sec

**Comment by n3oney** [comment]
i dont think it will run without bash as its in the shebang

**Comment by n3oney** [comment]
so no need to check that, just jq

**Comment by lwertzui** [comment]
there it is, it now checks dependecies before installing and breaks if one of them (most likely jq i'd say) isnt installed

**Comment by lwertzui** [comment]
yeah idk maybe if someone exclusively uses zsh or fsh or something

**Comment by lwertzui** [comment]
lets just be safe rather than sorry and leave that like it is

**Comment by lwertzui** [comment]
soo, merge now?

**Comment by lwertzui** [comment]
Plz?

File: README.md [hunk 1]
```diff
@@ -11,20 +11,23 @@ A simple bash script to fix Discord wanting to update to a version that isn't in
...
+curl -sSL https://raw.githubusercontent.com/n3oney/discord-update-skip/stable/set-config.sh | bash
+```
 
 ## Usage
 To patch your Discord configuration, you can simply run the script.
 
 ```sh
 discord-update-skip
 ```
+If you are on any other Distro, it is automatically run by using the install script.
```
Comment (n3oney) [inline README.md:25]: ran*

**Comment by lwertzui** [comment]
merge pllz?

**Comment by n3oney** [comment]
correct the spelling mistake please <3

**Comment by lwertzui** [comment]
Both are possible and plausible, if you really want it your way merge and then change it

**Comment by lwertzui** [comment]
In a passive construction like this run is the only correct one

**Pull request merged**